spec/active_record/connection_adapters/mysql2_ghost_adapter_spec.rb

```diff
@@ -4,17 +4,21 @@
 RSpec.describe ActiveRecord::ConnectionAdapters::Mysql2GhostAdapter do
   let(:logger) { double(:logger, puts: true) }
   let(:mysql_client) { double('Mysql2::Client', query_options: {}, query: nil) }
+  let(:table) { :foo }
+  let(:column) { :bar_id }
 
   subject { described_class.new(mysql_client, logger, {}, {}) }
 
   before do
     allow(mysql_client).to receive(:server_info).and_return({ id: 50_732, version: '5.7.32-log' })
+    if Gem.loaded_specs['activerecord'].version < Gem::Version.new('6.1')
+      allow(mysql_client).to receive(:escape).with(table.to_s).and_return(table.to_s)
+      allow(mysql_client).to receive(:more_results?)
```

Comment:
just mocked these because tests started failing on them.  I didn't do too much digging into how ActiveRecord is using these return values (other than obviously using the `escape` method to... escape a string)